### PR TITLE
39708 Customize and edit all model columns in rails admin

### PR DESCRIPTION
**Pull request by NgengChhengkim**

### CODE REVIEW

File: app/helpers/rails_admin/application_helper.rb [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                wording_for(:breadcrumb, a, am, o)
+              end
+            end
+            crumb
+          end
+        end.reverse.join.html_safe
+      end
+    end
+
+    # parent => :root, :collection, :member
```
Comment (minhld1709) [inline app/helpers/rails_admin/application_helper.rb]: dont need commend :)

**Comment by minhld1709** [comment]
OK

File: app/helpers/rails_admin/application_helper.rb [hunk 2]
```diff
@@ -0,0 +1,180 @@
+require "rails_admin/i18n_support"
+
+module RailsAdmin
+  module ApplicationHelper
+    include RailsAdmin::I18nSupport
+
+    def capitalize_first_letter(wording)
```
Comment (hellovietnam93) [inline app/helpers/rails_admin/application_helper.rb]: remove ()

File: app/models/concerns/rails_admin_course.rb [hunk 3]
```diff
@@ -2,18 +2,17 @@ module RailsAdminCourse
...
-        field :subjects
-      end
       list do
         field :name
         field :description
         field :status
       end
+
+      edit do
+          field :name
```
Comment (hellovietnam93) [inline app/models/concerns/rails_admin_course.rb]: ident

File: app/views/rails_admin/main/index.html.haml [hunk 4]
```diff
@@ -0,0 +1,159 @@
+:ruby
+  require 'rails_admin/config/fields/types/datetime.rb'
```
Comment (hellovietnam93) [inline app/views/rails_admin/main/index.html.haml]: using " instead  of '

**Comment by hellovietnam93** [comment]
Ok

**Pull request merged**